File: comms/push-notifications/device-registrations/delete-device-registration/delete-device-registration.ts
import { TwilioClient } from "twilio-comms";

async function main() {
    const client = new TwilioClient({
        accountId: "<username>",
        authToken: "<password>",
    });
    await client.pushNotifications.deviceRegistrations.delete("comms_device_registration_01h9krwprkeee8fzqspvwy6nq8");
}
main();

File: comms/push-notifications/device-registrations/delete-device-registration/delete-device-registration.py
from twilio import TwilioComms

client = TwilioComms(
    account_id="<username>",
    auth_token="<password>",
)

client.push_notifications.device_registrations.delete(
    device_registration_id="comms_device_registration_01h9krwprkeee8fzqspvwy6nq8",
)
